tests/pytests/common.py

```diff
@@ -857,7 +857,11 @@ def set_tight_maxmemory_for_oom(env, memory_limit_per = 0.8):
     # Get current memory consumption value
     memory_usage = env.cmd('INFO', 'MEMORY')['used_memory']
     # Set memory limit to less then memory limit
-    env.expect('config', 'set', 'maxmemory', int(memory_usage*(1/(memory_limit_per-0.01)))).ok()
+    required_memory = memory_usage * (1/memory_limit_per)
+    # Round up and add 1
+    new_memory = int(required_memory) + bool(required_memory%1) + 1
+
+    env.expect('config', 'set', 'maxmemory',new_memory).ok()
```

Comment:
Is this the flaky part? I believe that some internal redis stuff can consume memory so the new limit may not be enough. Maybe there is a way to ensure that? or should we relax the validation if that's the case?